src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutBase.cs

```diff
@@ -289,7 +291,11 @@ protected internal virtual void Open()
 			_popup?.SetValue(Popup.DataContextProperty, this.DataContext, precedence: DependencyPropertyValuePrecedences.Local);
 
 		partial void OnTemplatedParentChangedPartial(DependencyPropertyChangedEventArgs e)
+			=> SynchronizeTemplatedParent();
+
+		private void SynchronizeTemplatedParent()
 		{
+			var tag = (TemplatedParent as FrameworkElement)?.Tag;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'tag'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5577163)

Reply:
Yes, I agree with the robot on this one.